packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -434,13 +470,24 @@ class IOSDevice extends Device {
     bool ipv6 = false,
     String? userIdentifier,
     @visibleForTesting Duration? discoveryTimeout,
+    @visibleForTesting
+    ShutdownHooks? shutdownHooks,
   }) async {
     String? packageId;
     if (isWirelesslyConnected &&
         debuggingOptions.debuggingEnabled &&
         debuggingOptions.disablePortPublication) {
       throwToolExit('Cannot start app on wirelessly tethered iOS device. Try running again with the --publish-port flag');
     }
+
+    // Used in CI to test XcodeDebug workflow from older versions of iOS and Xcode.
```

Comment:
what is "older"? can you be more explicit?

Reply:
is it because we don't have iOS 17 yet in our CI?

Reply:
Yes, this is a workaround so we can test the Xcode debug flow in our CI prior to upgrading

Reply:
Can you add a comment there? isn't very clear from the code.

Reply:
Actually, add a TODO comment there in case we forget.